docs/meshstack.metadata-tags.md

```diff
@@ -187,8 +187,8 @@ The following extra metadata can be used in such a tag definition configuration:
 | `${workspaceIdentifier}`      | The project's workspace identifier                                                                                                                              |
 | `${workspaceName}`            | The projects's workspace name                                                                                                                                   |
 | `${projectName}`              | The project display name                                                                                                                                        |
-| `${landingzone}`              | The name of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the project does not have a Landing Zone applied       |
-| `${landingzoneIdentifier}`    | The identifier of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the project does not have a Landing Zone applied |
+| `${landingZoneName}`          | The name of the applied [Landing Zone](meshcloud.landing-zones.md). It contains `no-landingzone` in case the project does not have a Landing Zone applied       |
```

Comment:
The first sentence does not seem like a grammatically correct sentence

Reply:
I cant find any issue with the first sentence. Upon suggestion from Grammarly I changed it to `It contains `no-landingzone` if the project does not have a landing zone applied`.